src/main/java/de/tum/in/www1/artemis/domain/iris/settings/IrisChatSubSettings.java

```diff
@@ -34,6 +34,19 @@ public class IrisChatSubSettings extends IrisSubSettings {
     @Column(name = "rate_limit_timeframe_hours")
     private Integer rateLimitTimeframeHours;
 
+    @Nullable
+    @Column(name = "lecture_chat")
+    private Boolean lectureChat;
```

Comment:
Is it needed for this to be nullable? Usually you should avoid nullable booleans